dangerzone/gui/main_window.py

```diff
@@ -9,7 +9,15 @@
 from typing import List, Optional
 
 from colorama import Fore, Style
-from PySide2 import QtCore, QtGui, QtWidgets
+
+try:
+    from PySide6 import QtCore, QtGui, QtWidgets
+
+    # Alias renamed Qt6 methods for backwards compatibility
+    QtCore.QRegExp = QtCore.QRegularExpression
```

Comment:
In their [docs](https://doc-snapshots.qt.io/qt6-dev/qtcore-changes-qt6.html#regular-expression-classes), they propose using `QRegularExpression` in Qt5, as a migration path towards Qt6. I propose we do the same.

From a cursory glance on the sole regular expression that we use, I think we will not be affected by this change.

Reply:
Good catch